recipes/access.rb

```diff
@@ -60,3 +60,16 @@
   }]
   action :create
 end
+
+return unless node['platform_family'] == 'windows'
+
+if node['windows_hardening']['smbv1']['disable'] == true
```

Comment:
Hey @artem-sidorenko! So I thought long and hard about this. FC023 is no longer enforced by food-critic (modern versions) and I copied the style used in https://github.com/dev-sec/chef-windows-hardening/blob/master/recipes/rdp.rb#L9.